kuma/core/tests/__init__.py

```diff
@@ -62,3 +64,42 @@ def assertFileNotExists(self, path):
 
 class KumaTestCase(KumaTestMixin, TestCase):
     pass
+
+
+def call_on_commit_immediately(test_method):
+    """Useful for TestCase test methods, that ultimately depends on
+    `transaction.on_commit()` being called somewhere in the stack. These
+    would normally be not executed when TestCase rolls back.
+    But if the test wants to assert that something inside a
+    `transaction.on_commit()` is called, you're out of luck.
+    That's why this decorator exists. For example:
+
+        # In views.py
+
+        @transaction.atomic
+        def do_something(request):
+            transaction.on_commit(lambda x: do_other_thing)
+            return http.HttpResponse('yay!')
+
+
+        # In test_something.py
+
+        class MyTests(TestCase):
+
+            @call_on_commit_immediately
+            def test_something(self):
+                self.client.get('/do/something')
+
+    In this example, without the decorator the
```

Comment:
Nit: the comment is cut-off mid-sentence. Did you want to add more?

Reply:
Fixed in a followup.